### PR TITLE
adjust README module dependency example

### PR DESCRIPTION
**Pull request by gvaradarajan**
The previous line was not working for some reason when used in an actual project. This new way is more canonical to Cargo anyway I believe.

### CODE REVIEW

File: examples/modular-drivers/README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ index 79fbc5c..949b5ab 100644
  smol = "1.2"
  futures-lite = "1.12.0"
  micro-rdk = {version = "0.0.3", git = "https://github.com/viamrobotics/micro-rdk.git", features = ["esp32"]}
-+micro-rdk-modular-driver-example = { git = "https://github.com/viamrobotics/micro-rdk/tree/main/examples/modular-drivers" }
```
Comment (mattjperez) [inline examples/modular-drivers/README.md:38]: does it need the `package` specification? I figured the workspace would direct it if the name is the same as the intended package

**Review by mattjperez — APPROVED** [review]
LGTM, non-blocking question/suggestion

**Review by acmorrow — APPROVED** [review]
LGTM

File: examples/modular-drivers/README.md (hunk 1 above)
Comment (gvaradarajan) [reply examples/modular-drivers/README.md:38]: it doesn't but I think it's better communication to a reader about how it actually works to include it

**Pull request merged**